docs/5.community/6.roadmap.md

```diff
@@ -58,3 +58,32 @@ Auth           | Planned             | 3.x          | `nuxt/auth` to be announce
 Image          | Active              | 2.x and 3.x  | [nuxt/image](https://github.com/nuxt/image) | Nuxt 3 support is in progress: [nuxt/image#548](https://github.com/nuxt/image/discussions/548)
 Telemetry      | Active              | 2.x and 3.x  | [nuxt/telemetry](https://github.com/nuxt/telemetry/) | Nuxt 3 is supported. Stats to be public soon!
 I18n           | Active              | 2.x and 3.x  | [nuxt-modules/i18n](https://github.com/nuxt-modules/i18n) | See [nuxt-modules/i18n#1287](https://github.com/nuxt-modules/i18n/discussions/1287) for Nuxt 3 support
+
+## 📅 Long time support (LTS) versions
+
+### Current LTS version
+
+The current LTS is [V3.5](https://github.com/nuxt/nuxt/releases/tag/v3.5.3)
+
+### Next LTS version
+
+The next LTS is [V3.6](https://github.com/nuxt/nuxt/releases/tag/v3.6.1)
+
+Version number                                           | Status      | Initial release | Active LTS Start | Maintenance LTS Start | End Of Life  | Vue version | Minimal node version
+---------------------------------------------------------|-------------|-----------------|------------------|-----------------------|--------------|-------------|---------------------
+[V3.2](https://github.com/nuxt/nuxt/releases/tag/v3.2.3) | Unsupported | 02-09-2022      | 02-28-2022       | 03-14-2022            | 04-02-2022   | V3.2        | V16
+[V3.3](https://github.com/nuxt/nuxt/releases/tag/v3.3.3) | Maintenance | 03-14-2022      | 04-02-2022       | 04-11-2022            | 04-28-2022   | V3.2        | V16
+[V3.4](https://github.com/nuxt/nuxt/releases/tag/v3.4.3) | Maintenance | 04-11-2022      | 04-28-2022       | 05-16-2022            | 06-06-2022   | V3.2        | V16
+[V3.5](https://github.com/nuxt/nuxt/releases/tag/v3.5.3) | Current     | 05-16-2022      | 06-06-2022       | 06-23-2022            | 07-14-2022   | V3.3        | V16
+[V3.6](https://github.com/nuxt/nuxt/releases/tag/v3.6.1) | Active      | 06-23-2022      | 07-14-2022       | 07-28-2022            | 08-09-2022   | V3.3        | V16
+V3.7                                                     | Scheduled   | 07-28-2022      | 08-07-2022       | 08-21-2022            | 09-04-2022   | ~           | V18
```

Comment:
As a non-native English speaker, I prefer European date format (i.e. DD-MM-YYYY), because American date format is confusing when DD part is less than 13:

I can determine `04-12-2023` is `April 12th 2023` or `4th December 2023` only by checking other date strings in the same page.

Using word instead of numbers for months would be an alternative: `4th Dec. 2023` or `Dec. 4th 2023`.

Please ignore this comment if Nuxt docs uses American style date format in other places already and you just followed it.

![image](https://github.com/nuxt/nuxt/assets/16436160/b0a72a4d-481d-41b1-93ec-05900c4bc693)
_Image by https://iso.mit.edu/americanisms/date-format-in-the-united-states/_

Reply:
What about using YYYY-MM-DD? - I think this should be equally understandable for both Americans and rest of world.

Reply:
That is also fine. 👍 

My points is that it is beneficial for more people to use more intuitive (less confusing) date format and to use it consistently in Nuxt documentation (i.e. using other than American style date format).

Reply:
Completely agree 👍

Reply:
The next two days I will be a bit busy, but I will add the changes on Wednesday (Berlin time).

Reply:
I have been quite, bussy the last 3 weeks.
I have have updated the placeholder data.
Hopefully someone from the core team will give it the correct data.

Reply:
> What about using YYYY-MM-DD? - I think this should be equally understandable for both Americans and rest of world.

Should this be implemented every where?

I try checking the docs repo any use of the following format `\d{2}-\d{2}-\d{4}` in VSCode,
But I did not find any other use of it, so could this be the only place this date format is used?